pkg/cmd/cli/cmd/importimage.go

```diff
@@ -26,12 +26,21 @@ import (
 
 var (
 	importImageLong = templates.LongDesc(`
-		Import tag and image information from an external Docker image repository
+		Import the latest images from a Docker registry into an image stream
```

Comment:
It's not about not saying about IS, it's about not making them believe that `oc import-image` imports the entire image, that's very frequent misunderstanding I struggle with. We need to be clear that we *only* import image metadata. Stating we import image means for 99% of users we import the image into our registry, that's from my experience talking to users.